### DISCUSSION TITLE
Request programming resources you want

### DISCUSSION THREAD

**Discussion by Obscurely** [Obscurely/falion#15] [Ideas]
Request programming resources that you'd want to see first party support for. Based on which are requested the most and which I think should be added I will either choose and create a new issue or first make a poll with multiple and vote which you want first.

**Comment by DavidVVLL** [comment]
will you be continously working on this project?

**Reply by Obscurely** [reply]
I do have a lot things on hand right now so I am gonna work on it like 2 hours a week atm, maybe more. But I will be consistently working on it at least. And actually maybe more I don't know I haven't planed everything I have yet.
